build-logic/src/main/kotlin/org.sonarsource.text.rule-api.gradle.kts

```diff
@@ -0,0 +1,83 @@
+val rulApiVersion = "2.7.0.2612"
+
+repositories {
+  maven {
+    url = project.uri("https://repox.jfrog.io/repox/sonarsource-private-releases")
+    authentication {
+      credentials {
+        val artifactoryUsername: String? by project
+        val artifactoryPassword: String? by project
+        username = artifactoryUsername
+        password = artifactoryPassword
+      }
+    }
+  }
+}
+
+val ruleApi = configurations.create("ruleApi")
+
+dependencies {
+  ruleApi("com.sonarsource.rule-api:rule-api:${rulApiVersion}")
+}
+
+val ruleApiUpdateSecrets = tasks.register<Exec>("ruleApiUpdateSecrets") {
```

Comment:
Can be probably rewritten with `JavaExec` instead of `Exec`

Reply:
Yes, maybe